v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/spannerchangestreamstobigquery/FailsafeModJsonToTableRowTransformer.java

```diff
@@ -181,7 +183,7 @@ public void setUp() {
                   "Caught exception when setting up FailsafeModJsonToTableRowFn, message: %s,"
                       + " cause: %s",
                   Optional.ofNullable(e.getMessage()), e.getCause()));
-          seenException = true;
```

Comment:
Because we still want the first error message to be printed at later stage when process the element [here](https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/7a8f5ddd6879bc5de77dbba4e41f8bbbd225cddd/v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/spannerchangestreamstobigquery/FailsafeModJsonToTableRowTransformer.java#L211).